packages/cli-poa/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@percy/cli-poa",
```

Comment:
we will need to change the package name. Commenting it to keep it in check.

Reply:
have sdk-utils package as a reference point 
https://github.com/percy/cli/tree/master/packages/sdk-utils